packages-next/types/src/admin-meta.ts

```diff
@@ -5,7 +5,7 @@ export type FieldControllerConfig<FieldMeta extends JSONValue | undefined = unde
   listKey: string;
   path: string;
   label: string;
-  customViews: Record<string, any>;
+  customViewsHash: Record<string, any>;
```

Comment:
This should just be customViews, it's not the hash

Reply:
Nice catch, overcooked the search and replace :-)